api/admin/controller.py

```diff
@@ -631,3 +631,61 @@ def circulation_events(self):
         }, results)
 
         return dict({ "circulation_events": events })
+
+    def bulk_circulation_events(self):
+        annotator = AdminAnnotator(self.circulation)
```

Comment:
It looks like you don't use this Annotator (which makes sense since it's for OPDS feeds).